lib/media/ttml_text_parser.js

```diff
@@ -217,6 +208,25 @@ shaka.media.TtmlTextParser.getLeafNodes_ = function(element) {
 
 
 /**
+ * Insert \n where <br> tags are found
+ *
+ * @param {!Node} element
```

Comment:
We solved this elsewhere in with an assertion and a cast:

    if (childNodes[i].nodeType == Node.ELEMENT_NODE && ...) {
      goog.asserts.assert(childNodes[i] instanceof Element,
                          'Node should be Element!');
      var leafChildren = shaka.media.TtmlTextParser.getLeafNodes_(
          /** @type {Element} */(childNodes[i]));